client/tailscale/localclient.go

```diff
@@ -1019,6 +1019,23 @@ func (lc *LocalClient) DebugDERPRegion(ctx context.Context, regionIDOrCode strin
 	return decodeJSON[*ipnstate.DebugDERPRegionReport](body)
 }
 
+func (lc *LocalClient) StartDebugCapture(ctx context.Context, addr string) error {
```

Comment:
godoc. and what is "addr" ?

I'd like to be able to attach to multiple point(s) inside the data plane, like all-in, post-filter-in, out, etc.

Reply:
Why attach separately? I've always just fed wireshark the firehose and used its filtering.

In this case, our custom dissector exposes a field `tsdebug.path` which we could use to zoom in on only one of the points.

Reply:
how does wireshark know what we rejected?

Reply:
Wireshark knows whatever we tell it. For something like filter-dropped packets, we could call the callback after the filter and add a new metadata field `dropped` or something like that.

Currently the cb is wired before ACL eval, cuz WIP.